pkgs/development/tools/build-managers/msbuild/default.nix

```diff
@@ -33,13 +33,19 @@ stdenv.mkDerivation rec {
   ];
 
   # https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=msbuild
-  phases = ["unpackPhase" "buildPhase" "installPhase" "installCheckPhase"];
+  phases = ["unpackPhase" "patchPhase" "buildPhase" "installPhase" "installCheckPhase"];
```

Comment:
I am not very happy that it overwrites phases. Can this be removed and the normal phases be used? Maybe a different PR?

Reply:
It seems to be fine without it, so I removed it in its own commit.